src/orca-loader/Elements.res

```diff
@@ -4,6 +4,7 @@ open Identity
 open Utils
 open EventListenerManager
 open ApplePayTypes
+@send external alert: ('t, string) => unit = "alert"
```

Comment:
revert unintended changes from 1f754b6f4023709752ef892c0e2861d34aa4f2fe

Reply:
reverted

Reply:
Still seeing this

Reply:
updated